docs/client/full-api/api/templates.md

```diff
@@ -97,6 +120,16 @@ These callbacks fire once and are the first group of callbacks to fire.
 Handling the `created` event is a useful way to set up values on template
 instance that are read from template helpers using `Template.instance()`.
 
+```javascript
+Template.myPictures.onCreated(function () {
+  // set up local reactive variables
+  this.dataVar = new ReactiveVar("some value");
```

Comment:
I think the examples will be much better if we use real-life situations, rather than generic names like "foo", "bar", and "dataVar".